controllers/theatre.controller.js

```diff
@@ -97,3 +99,58 @@ exports.getSingleTheatre = async (req,res)=>{
    }
 
 }
+
+exports.getallMoviesByTheatreId = async (req,res)=>{
+
+    try{
+        const theatre = await Theatre.findOne({_id : req.params.id});
+        const movieIds = theatre.movies;
+
+        const movies = await Movie.find({_id : movieIds})
+        res.status(200).send(movies);
+    
+    }catch(err){
+        console.log("#### Error while getting all movie of that theatre ####", err.message);
+        res.status(500).send({
+            message : "Internal server error while getting the theatre"
+        })
+    }
+ 
+}
+
+exports.addingAndRemovingMovie = async (req,res)=>{
+
+    try{
+        const theatre = await Theatre.findOne({_id : req.params.id});
+        const movieIds = req.body.movies;
+
+        if(req.body.operationType == constants.operationType.add){
```

Comment:
We should support both the functionality of addition and removal in the single API call